lib/take_away.rb

```diff
@@ -0,0 +1,48 @@
+require_relative './menu'
+require_relative './send_sms'
+
+class TakeAway
+  attr_reader :cart
+
+  def initialize(menu: Menu.new, sender: SmsSender.new)
+    @menu = menu
+    @sender = sender
+    @cart = []
+  end
+
+  def menu
+    @menu.list_dishes
+  end
+
+  def add_to_cart(name, quantity = 1)
+    new_dish = @menu.select_dish(name, quantity)
+    update_dish_quantity(new_dish) || @cart << new_dish
+  end
+
+  def update_dish_quantity(new_dish)
+    edit = false
```

Comment:
I'm not clear on what this edit variable is for? Could a more descriptive word be used perhaps?